beacon-chain/blockchain/service_test.go

```diff
@@ -97,7 +97,7 @@ func setupBeaconChain(t *testing.T, beaconDB db.Database) *Service {
 		WithAttestationPool(attestations.NewPool()),
 		WithSlashingPool(slashings.NewPool()),
 		WithExitPool(voluntaryexits.NewPool()),
-		WithP2PBroadcaster(&mockBroadcaster{}),
+		WithP2PBroadcaster(&mockAccesser{}),
```

Comment:
```suggestion
		WithP2PBroadcaster(&mockAccessor{}),
```

I think the correct spelling is like this

Reply:
Fixed in 0f2b6679c121ee199229d0206b40c71a7b21a890.

(Not accepting your suggestion as is because `mockAccesser` needs to be modified in some other places in the code as well.)